src/test/resources/TestNG.xml

```diff
@@ -2,10 +2,11 @@
 <!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
 
 <suite name="TestNG">
-
+<parameter name="baseUrl" value="https://bbb.testpro.io/"/>
     <test name="Example Test" preserve-order="false">
         <classes>
-            <class name="CucumberRunner"/>
+            <class name="HomeWork19"/>
+            <class name="BaseTest"/>
```

Comment:
The code is not compiled! Please finish the homework!You got this problem, please correct it. 

![image](https://user-images.githubusercontent.com/104761482/226828436-d5fefd72-3222-4620-97cf-ce6684928486.png)